docs/Measuring_Resonances.md

```diff
@@ -99,7 +141,10 @@ Afterwards, check and follow the instructions in the
 Make sure the Linux SPI driver is enabled by running `sudo
```

Comment:
I think this sentence 'Make sure the Linux SPI driver is enabled by running...' can be moved at the start of the next section for adxl345. Then it will follow the same logic as the recommendation to enable I2C with the MPU chip